### DISCUSSION TITLE
Add config for glide server

### DISCUSSION THREAD

**Discussion by lotarbo** [awcodes/filament-curator#111] [Ideas]
Hello, i was thinking about simple config file for glider server without using facade.
Ex: if we need `imagick` driver against `gd`, much simply to edit config file

**Comment by awcodes** [comment]
Honestly, I'm not sure a config for the glide server would be beneficial since it would be easier to just implement your own server. Also, I'm trying to stay more aligned to the changes coming with Filament v3's context provider. I do think it could be beneficial though to have a setting for switching out gd, so I will look into that.

**Comment by awcodes** [comment]
With the v2.2.2 release you can now set the driver via `Curator::glideDriver('imagick')` in your Service Provider.

**Reply by lotarbo** [reply]
thank you
